lib/src/room.dart

```diff
@@ -618,6 +618,14 @@ class KurentoRoom extends ChangeNotifier implements Room {
           // Agent-initiated end message, but the impact of that is low (the Explorer can end the call themselves).
           _getServiceRequestStatusTimer?.cancel();
 
+          final bool hasOutgoingVideoSenderTrack = _connectionByTrackId[_localTrackId!]?.ownsVideoTrack != true;
```

Comment:
Is this supposed to be `== true`?

Reply:
Good catch! This came with a name change for clarity but I forgot to change the condition.